models/adminInvitation.js

```diff
@@ -0,0 +1,16 @@
+const { Schema, model } = require('mongoose')
+
+const adminInvitationSchema = new Schema({
+  email: {
+    type: String,
+    required: true,
+    unique: true,
+    lowercase: true
+  },
+  dateOfInvitation: {
+    type: Date,
+    required: true
```

Comment:
shouldn't it be here a default value instead?

Reply:
ok